config/test_cases/test_pull_request_openshift_ansible_tox.yml

```diff
@@ -18,6 +18,20 @@ actions:
     repository: "openshift-ansible"
     script: |-
       tox
+  - type: "script"
```

Comment:
I'd like to be able to start running these tests. How about we merge this and just agree not to actually merge my PR supplying the tests until we've either established confidence it won't break the build or decided to do something else.